app/models/manageiq/providers/vmware/infra_manager/inventory/parser/virtual_machine.rb

```diff
@@ -28,7 +30,7 @@ def parse_virtual_machine_summary(vm_hash, props)
       if summary_config
         uuid = summary_config[:uuid]
         unless uuid.blank?
-          vm_hash[:uid_ems] = MiqUUID.clean_guid(uuid) || uuid
+          vm_hash[:uid_ems] = clean_guid(uuid) || uuid
```

Comment:
@djberg96 I think we can just do `vm_hash[:uid_ems] = clean_guid(uuid)` without the `|| uuid` if the only two cases that method returns `nil` for is if the uuid is `nil` (in which case we'd get `nil || nil`) or if the string is empty which isn't a valid uid_ems anyway.  WDYT?

Reply:
@agrare I agree, will update.